processor/attributesprocessor/README.md

```diff
@@ -169,6 +176,18 @@ attributes:
       # This is an optional field.
       log_names: [<item1>, ..., <itemN>]
 
+      # The metric name must match at least one of the items.
+      # This is an optional field.
+      metric_names: [<item1>, ..., <itemN>]
+
+      # Expressions are used to match metrics. A match occurs if the expression matches at least
+      # one data point within a metric.
+      expressions: [<item1>, ..., <itemN>]
+
+      # Resource attributes defines a list of possible resource attributes to match metrics against.
+      # A match occurs if any resource attribute matches all expressions in this given list.
+      resource_attributes: [<item1>, ..., <itemN>]
```

Comment:
Additional problem with this parameter is that it works only for metrics but is named like a universal one

Reply:
I've removed this in favor of using the pre-existing resource attribute.